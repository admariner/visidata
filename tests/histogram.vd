sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
global		numeric_binning	set-option	True		
benchmark	Quantity		type-int		#	
benchmark	Quantity		addcol-expr	Quantity > 1	=	
benchmark	Quantity > 1		select-col-regex	True	|	
benchmark			dup-selected		"	
benchmark_selectedref	Quantity		freq-col		F	
benchmark_selectedref_Quantity_freq	histogram		hide-col		-	Hide current column
