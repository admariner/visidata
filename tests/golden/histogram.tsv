Quantity	count	percent
2.00 - 30.40	26	92.86
30.40 - 58.80	1	3.57
58.80 - 87.20	0	0.00
87.20 - 115.60	0	0.00
115.60 - 144.00	1	3.57
